contracts/Account.cairo

```diff
@@ -47,11 +46,8 @@ end
 
 @view
 func assert_only_self{
-        storage_ptr: Storage*,
-        pedersen_ptr: HashBuiltin*,
-        syscall_ptr: felt*,
-        range_check_ptr
-    }():
+        syscall_ptr : felt*, pedersen_ptr : HashBuiltin*,
+        range_check_ptr} ():
```

Comment:
I kindof liked the previous formating, it makes it easier to read and to edit (here and other places).
 What do you think?

Reply:
I definitely agree, it's much easier to read. It seemed safer to follow Starknet's documentation examples, but I'd be happy to reformat